sdks/python/apache_beam/__init__.py

```diff
@@ -66,10 +66,10 @@
 import sys
 
 
-if sys.version_info.major != 2:
+if sys.version_info < (2, 7):
```

Comment:
This would not work with python 3. For example (3, 0) > (2, 7) but we would like to show the error in that case.